e2e-polybft/e2e/aa_test.go

```diff
@@ -0,0 +1,108 @@
+package e2e
+
+import (
+	"fmt"
+	"math/big"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi/artifact"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+)
+
+func TestE2E_AccountAbstraction(t *testing.T) {
+	const (
+		amount             = 15
+		customSecretPrefix = "other-"
+	)
+
+	someRandomAddress1 := types.StringToAddress("0xff00ff00ff00cc00bb11893")
+	someRandomAddress2 := types.StringToAddress("0xff00ff00ff00cc00bb11894")
+
+	deployerAccount, err := wallet.GenerateAccount()
+	require.NoError(t, err)
+
+	aaInvoker, err := artifact.DecodeArtifact([]byte(contractsapi.AccountAbstractionInvokerArtifact))
+	require.NoError(t, err)
+
+	cluster := framework.NewTestCluster(t, 4,
+		framework.WithSecretsCallback(func(_ []types.Address, clst *framework.TestCluster) {
+			addresses, err := clst.InitSecrets(customSecretPrefix, 2) // generate two additional accounts
+			require.NoError(t, err)
+
+			userBalance := fmt.Sprintf("%s:%d", addresses[0], amount*3)
+			relayerBalance := fmt.Sprintf("%s:%d", addresses[1], 0xFF00FF00000000)
+			deployerBalance := fmt.Sprintf("%s:%d", deployerAccount.Ecdsa.Address(), 0xFF00FF00000000)
```

Comment:
Suggesting this, because it is more understandable what is the premined amount (you can use `ethgo.Gwei` as well if needed).
```suggestion
			relayerBalance := fmt.Sprintf("%s:%d", addresses[1], ethgo.Ether(1))
			deployerBalance := fmt.Sprintf("%s:%d", deployerAccount.Ecdsa.Address(), ethgo.Ether(1))
```